Linux Script/configure-linux.sh

```diff
@@ -89,735 +90,815 @@ LINUX_ENV_VALIDATED="false"
-	checkIfMinVersionOfRsyslog
-
-	#check if selinux service is enforced. if yes, ask the user to manually disable and exit the script
-	checkIfSelinuxServiceEnforced
-	
-	#update rsyslog.conf and adds $MaxMessageSize in it
-	modifyMaxMessageSize
-
-	LINUX_ENV_VALIDATED="true"
+        #check if the user has root permission to run this script
+        checkIfUserHasRootPrivileges
+  
+        #check if the OS is supported by the script. If no, then exit
+        checkIfSupportedOS
+  
+        #set the basic variables needed by this script
+        setLinuxVariables
+  
+        #check if the Loggly servers are accessible. If no, ask user to check network connectivity & exit
+        checkIfLogglyServersAccessible
+  
+        #check if user credentials are valid. If no, then exit
+        checkIfValidUserNamePassword
+  
+        #get authentication token if not provided
+        getAuthToken
+  
+        #check if authentication token is valid. If no, then exit.
+        checkIfValidAuthToken
+  
+        #checking if syslog-ng is configured as a service
+        checkifSyslogNgConfiguredAsService
+  
+        #check if systemd is present in machine.
+        checkIfSystemdConfigured
+  
+        #check if rsyslog is configured as service. If no, then exit
+        checkIfRsyslogConfiguredAsService
+  
+        #check if multiple rsyslog are present in the system. If yes, then exit
+        checkIfMultipleRsyslogConfigured
+  
+        #check for the minimum version of rsyslog i.e 5.8.0. If no, then exit
+        checkIfMinVersionOfRsyslog
+  
+        #check if selinux service is enforced. if yes, ask the user to manually disable and exit the script
+        checkIfSelinuxServiceEnforced
+  
+        #update rsyslog.conf and adds $MaxMessageSize in it
+        modifyMaxMessageSize
+  
+        LINUX_ENV_VALIDATED="true"
 }
 
 # executing the script for loggly to install and configure rsyslog.
 installLogglyConf()
 {
-	#log message indicating starting of Loggly configuration
-	logMsgToConfigSysLog "INFO" "INFO: Initiating Configure Loggly for Linux."
-
-	if [ "$LINUX_ENV_VALIDATED" = "false" ]; then
-		checkLinuxLogglyCompatibility
-	fi
-
-	#if all the above check passes, write the 22-loggly.conf file
-	checkAuthTokenAndWriteContents
-
-	#create rsyslog dir if it doesn't exist, Modify the permission on rsyslog directory if exist on Ubuntu
-	createRsyslogDir
-
-	if [ "$LINUX_DO_VERIFICATION" = "true" ]; then
-		#check if the logs are going to loggly fro linux system now
-		checkIfLogsMadeToLoggly
-	fi
-
-	if [ "$IS_INVOKED" = "" ]; then
-		logMsgToConfigSysLog "SUCCESS" "SUCCESS: Linux system successfully configured to send logs via Loggly."
-	fi
-
+        #log message indicating starting of Loggly configuration
+        logMsgToConfigSysLog "INFO" "INFO: Initiating Configure Loggly for Linux."
+    
+        if [ "$LINUX_ENV_VALIDATED" = "false" ]; then
+        checkLinuxLogglyCompatibility
```

Comment:
@Shwetajain148:  This statement should appear inside the IF block. Make the same change at all the places where this kind of indentation problem is present.